### PR TITLE
Create Blip “gcp-artifact-registry”

### PR DESCRIPTION
**Pull request by arnaud-fossop**
Automatically generated by Decap CMS

### CODE REVIEW

**Review by julienblin — CHANGES_REQUESTED** [review]
@arnaud-fossop - I am not sure if there is strong value documenting this one.
What does GCP artifact registry does better / or worse than competing offerings?
e.g. - would you recommend using it even if you are not in GCP?
At a high-level it feels to me like it's something you kinda need to adopt if using artifacts in a GCP deployment.

Curious to here your thoughs.

File: src/content/blips/gcp-artifact-registry.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: GCP Artifact Registry
+description: GCP Artifact Registry is the new single place to manage all build
+  artifacts in GCP. It is notably replacing Container Registry for managing
+  Docker images. GCP Artifact Registry also offer the possibility to handle
```
Comment (lvancraen) [inline src/content/blips/gcp-artifact-registry.md:5]:
```suggestion
  Docker images. GCP Artifact Registry also offers the possibility to handle
```

File: src/content/blips/gcp-artifact-registry.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  build artifacts for Python (similar to a private PyPi), Java (private Maven)
+  and Node.js (private npm).
+opinion: >-
+  GCP Artifact Registry provides an artifact registry that is fully integrated
+  with GCP. This is a good choice for a startup building on GCP and benefitting
+  from the Google Startup Program.
+
+  GCP Artifact Registry allows to make artifacts available by region, to provide access based on company security policy. Different registries can also be created for different purposes (dev vs prod)
+
+  In Docker registry, images are scan for security vulnerabilities.
```
Comment (lvancraen) [inline src/content/blips/gcp-artifact-registry.md:15]:
```suggestion
  In Docker registry, images are scanned for security vulnerabilities.
```

**Comment by arnaud-fossop** [comment]
> @arnaud-fossop - I am not sure if there is strong value documenting this one. What does GCP artifact registry does better / or worse than competing offerings? e.g. - would you recommend using it even if you are not in GCP? At a high-level it feels to me like it's something you kinda need to adopt if using artifacts in a GCP deployment.
> 
> Curious to here your thoughs.

In my previous work experience, we were using a private PyPi for Python packages, an Enterprise Docker account for Docker images, and sometimes Gitlab repository for pre-release images. When we started Synapse, we also had the choice with Github Packages. But all these options were coming with an additional cost and additional overhead to manage authentications. So that's why I am insisting on the full integration and economic aspect of using GCP Artifact registry for users in GCP (as clearly stated in my conclusion in the opinion field), and at the same time reporting about my experience with it.

If, for you, the choice of GCP Artifact Registry when you are working with GCP is natural, I cannot argue with it 😄 . So, is it valuable?

File: src/content/blips/gcp-artifact-registry.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+title: GCP Artifact Registry
+description: GCP Artifact Registry is the new single place to manage all build
+  artifacts in GCP. It is notably replacing Container Registry for managing
```
Comment (jorge-vasquez-acn) [inline src/content/blips/gcp-artifact-registry.md:4]: I wonder if this tool is mature and has enough features to be considered a competitor for Nexus.  Say, I am not operating at all in GCP, is it worth it for me to install the Artifact Registry say if I am in AWS?  
That being said I find it very clever from google to merge in one place the container images and the libraries from build system tools: maven, npm, pip.  To my knowledge no other of the big cloud providers is doing this in one same place so kudos to GCP for thinking of this abstraction which is what actually Nexus does as well.

File: src/content/blips/gcp-artifact-registry.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
+  Other advantages of GCP Artifact Registry are that it allows to make artifacts available by region, to provide access based on company security policy. Different registries can also be created for different purposes (dev vs prod)
+
+  In the Docker registry, images are scanned for security vulnerabilities.
+
+
+  There are some constraints/limitations to account for when using GCP Artifact Registry:
+
+  * the Python registry cannot automatically mirror the official PyPi.org (like other private PyPi servers allow to)
```
Comment (jorge-vasquez-acn) [inline src/content/blips/gcp-artifact-registry.md:22]: But I guess that from a security point of view mirroring the whole registry is not such a great idea, the security team might want to make sure that whatever library is being targeted is fully secure before opening it to the development team.

Comment (arnaud-fossop) [reply src/content/blips/gcp-artifact-registry.md]: I added a not about the security aspect

**Comment by julienblin** [comment]
> > @arnaud-fossop - I am not sure if there is strong value documenting this one. What does GCP artifact registry does better / or worse than competing offerings? e.g. - would you recommend using it even if you are not in GCP? At a high-level it feels to me like it's something you kinda need to adopt if using artifacts in a GCP deployment.
> > Curious to here your thoughs.
> 
> In my previous work experience, we were using a private PyPi for Python packages, an Enterprise Docker account for Docker images, and sometimes Gitlab repository for pre-release images. When we started Synapse, we also had the choice with Github Packages. But all these options were coming with an additional cost and additional overhead to manage authentications. So that's why I am insisting on the full integration and economic aspect of using GCP Artifact registry for users in GCP (as clearly stated in my conclusion in the opinion field), and at the same time reporting about my experience with it.
> 
> If, for you, the choice of GCP Artifact Registry when you are working with GCP is natural, I cannot argue with it 😄 . So, is it valuable?

OK I get it now! This makes sense.
However, I don't get that level of clarity in the opinion in the blip. Could you rephrase so that this is better?
Don't get caught on generic formulation - we want clear opinions that are easy and fast to read/parse :-)

File: src/content/blips/gcp-artifact-registry.md (hunk 3 above)
Comment (arnaud-fossop) [reply src/content/blips/gcp-artifact-registry.md:4]: Nexus, Github Packages, Gitlab package registry and container registry are competitors.

**Review by jorge-vasquez-acn — APPROVED** [review]

**Review by julienblin — APPROVED** [review]
OK - thanks @arnaud-fossop for the changes!
Let's get that in!

**Pull request merged**